### PR TITLE
Upgrade proxy and server rename ke

### PR DESCRIPTION
**Pull request by keichwa**
- [x] Better first merge https://github.com/SUSE/doc-susemanager/pull/740

### CODE REVIEW

**Comment by keichwa** [comment]
argl, this also has the proxy-upgrade/update changes :-(

File: modules/upgrade/pages/proxy-update.adoc [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+1. Run a salt command: `spacewalk-proxy stop`
+2. Update the packages
+3. Run a salt command: `spacewalk-proxy start`
+////
+
+
+
+== Prepare to Update
+
+Before you perform any proxy upgrade, schedule a maintenance window.
```
Comment (juliogonzalez) [inline modules/upgrade/pages/proxy-update.adoc]: It should be "update" here.

File: modules/upgrade/pages/proxy-update.adoc [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+The clients registered to {productname} through the proxy will not be able to connect to {productname} while the upgrade is in progress.
```
Comment (juliogonzalez) [inline modules/upgrade/pages/proxy-update.adoc]: And here.

File: modules/upgrade/pages/proxy-update.adoc [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+. Select the packages to be updated on the proxy.
+
+. On the {productname} Server where the proxy is registered as a client, start the proxy service:
++
+----
+spacewalk-proxy start
+----
+
+If you need to update many proxies, you can create an action chain of this command sequence.
+You can use the action chain to perform upgrades on multiple proxies at the same time.
```
Comment (juliogonzalez) [inline modules/upgrade/pages/proxy-update.adoc]: And here :-)

**Comment by keichwa** [comment]
should be fine now.  please check again.

**Comment by keichwa** [comment]
On 2019-09-27 02:01, Lana Brindley wrote:
> Loquacity approved this pull request.

Thanks!  Hope we got it right ;-)

-- 
Karl Eichwalder

**Pull request merged**